src/eth/execution.ts

```diff
@@ -0,0 +1,87 @@
+import express from 'express';
+import { clients } from '@snapshot-labs/sx';
+import { wallet } from './dependencies';
+
+type Cache = {
+  [key: string]: {
+    timestamp: number;
+    receipt: any;
+  };
+};
+
+const CACHE_TIMEOUT = 5 * 60 * 1000;
+const CACHED_RECEIPTS: Cache = {};
+
+function clearCache() {
+  const now = Date.now();
+  Object.keys(CACHED_RECEIPTS).forEach(key => {
+    if (now - CACHED_RECEIPTS[key].timestamp > CACHE_TIMEOUT) {
+      delete CACHED_RECEIPTS[key];
+    }
+  });
+}
+
+const router = express.Router();
+const client = new clients.EvmEthereumTx();
+
+router.use((req, res, next) => {
+  clearCache();
+  next();
+});
+
+router.post('/execute', async (req, res) => {
```

Comment:
Ideally we should have a single route that is using JSONRPC and routing would work with `method` parameter